apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/MapLikeToContainInAnyOrderCreatorSamples.kt

```diff
@@ -6,6 +6,28 @@ import kotlin.test.Test
 
 class MapLikeToContainInAnyOrderCreatorSamples {
 
+    @Test
+    fun entry() {
+        expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.entry(2 to "b")
+
+        fails {
+            expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.entry(2 to "c")
+        }
```

Comment:
```suggestion
        fails { // because the value "b" of key 2 (which exists in the subject) is not equal to "c"
            expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.entry(2 to "c")
        }
```
Here you forgot to mention why it failed. I also propose an alternative formatting we used in other samples (although also totally fine to write it after the entry as you did further below. It's, just a suggestion)